client/pipeline_events.go

```diff
@@ -0,0 +1,18 @@
+package client
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"github.com/calyptia/api/types"
+)
+
+// CreatePipeline within an aggregator.
+// The pipeline name must be unique within the aggregator.
+// The resource profile must exist already. If you don't provide one, it will default to "best-effort-low-resource".
+// Use them to easily deploy configured agents to the aggregator.
+func (c *Client) CreatePipelineEvents(ctx context.Context, aggregagotID string, payload types.CreatePipelineEvents) (types.CreatePipelineEventsResponse, error) {
+	var out types.CreatePipelineEventsResponse
+	return out, c.do(ctx, http.MethodPost, "/v1/aggregators/"+url.PathEscape(aggregagotID)+"/pipeline/events", payload, &out)
```

Comment:
```suggestion
	return out, c.do(ctx, http.MethodPost, "/v1/pipelines/"+pipelineID+"/events", payload, &out)
```

With your way, you have to also pass the aggregator ID alongside the pipeline ID in the body too.

THe way we have been doing it is just pass the pipeline ID in the URL, and no pipeline ID in the body.

So I would remove the `PipelineID` from the requets body and move it to the URL.

Reply:
Oh, its an array...

Reply:
```suggestion
	return out, c.do(ctx, http.MethodPost, "/v1/aggregators/"+url.PathEscape(aggregagotID)+"/pipeline_events", payload, &out)
```
Then I change my suggestion to this.